package.json

```diff
@@ -2,18 +2,21 @@
   "name": "happiness-app",
   "version": "0.0.0",
   "scripts": {
-    "dev": "vite",
+    "dev": "vite --host",
```

Comment:
I'm not sure why it was added - I think that it could be removed, if you'd like to remove it?

It seems like it could be used so that you can access localhosts from an external device, like if I'm using vscode liveshare.
[link to thing i found](https://github.com/vitejs/vite/discussions/3396)